### DISCUSSION TITLE
Please tell me how to create article like this which can help users to copy code

### DISCUSSION THREAD

**Discussion by minimayatech** [h5bp/html5-boilerplate#2994] [Q&A] [answered]
![Screenshot_2023-08-15-10-12-51-21_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/h5bp/html5-boilerplate/assets/142289191/debc0b5f-e2d2-4bf9-8676-8d8ec4c237fb)

**Comment by roblarsen** [comment, marked as answer]
Ask Harry?

**Reply by minimayatech** [reply]
Bro do you no how to write article like this in Blog or website that can help users to copy programming code From article
